apps/client/src/components/Location/LocationResult.js

```diff
@@ -9,49 +9,14 @@ import Form from "../Form";
 import Nav from "../Nav";
 import RegisterLookupSummary from "../RegisterLookupSummary";
 
-const LocationResult = ({
-  isActive,
-  isFinished,
-  matomoTrackEvent,
-  setFinishedState,
-  setActiveState,
-  topic,
-}) => {
+const LocationResult = ({ matomoTrackEvent, setActiveState, topic }) => {
   const sessionContext = useContext(SessionContext);
   const address = sessionContext[topic.slug].address || {};
-  const { questionIndex } = sessionContext[topic.slug] || {};
   const hasSTTR = !!topic.sttrFile;
 
   const onSubmit = (e) => {
     e.preventDefault();
-    if (hasSTTR) {
-      if (typeof questionIndex !== "number") {
-        // Init checker by setting question index to 0
-        sessionContext.setSessionData([
-          topic.slug,
-          {
-            questionIndex: 0,
-          },
-        ]);
-      }
-
-      setFinishedState(sections.LOCATION_RESULT, true);
-
-      const eventSection = isFinished(sections.QUESTIONS)
-        ? sections.CONCLUSION
-        : sections.QUESTIONS;
-
-      matomoTrackEvent({
-        action: actions.CLICK_INTERNAL_NAVIGATION,
-        name: `${eventNames.FORWARD} ${eventSection}`,
-      });
-
-      if (isFinished(sections.QUESTIONS)) {
-        setActiveState(sections.CONCLUSION);
-      } else {
-        setActiveState(sections.QUESTIONS);
-      }
-    } else {
+    if (!hasSTTR) {
```

Comment:
LocationResult is only shown in the OLO flow, so this statement should be useless.